Makefile

```diff
@@ -24,3 +24,10 @@ build-image: version
 version:
 	$(eval VERSION=$(if $(OVERWRITE_VERSION),$(OVERWRITE_VERSION),v$(shell scripts/vertagen/vertagen.sh)))
 	@echo "turing version:" $$VERSION
+
+.PHONY: setup
+setup:
+	@echo "Setting up dev tools..."
+	@test -x pre-commit || pip install pre-commit
```

Comment:
Not sure if this works as intended:

```shell
➜  ~ test -x git || echo test
test
➜  ~ which git
/usr/local/bin/git
```

Reply:
Thanks for catching. I fixed it now using: `test -x "$(which pre-commit)" || pip install pre-commit`